src/includes/configuration/config-intro/android.mdx

```diff
@@ -19,6 +19,7 @@ public class MyApplication extends Application {
         options -> {
           // your Sentry project/dashboard
           options.setDsn("___PUBLIC_DSN___");
+          options.setRelease("io.example@1.1.0");
```

Comment:
Android does it automatically, you don't need that.

Reply:
Please remove @marandaneto, I just started this PR for others to add to. 

Also, the releases documentation does not mention this, can you please add a mention there like for Apple?

Reply:
oh just read the PR description, sry

Reply:
Thanks a bunch!

Reply:
removed

Reply:
added the mention to the release docs for Android too